src/coord/rmr/conn.c

```diff
@@ -423,7 +425,7 @@ static int MRConn_SendAuth(MRConn *conn) {
   CONN_LOG(conn, "Authenticating...");
 
   // if we failed to send the auth command, start a reconnect loop
-  if (redisAsyncCommand(conn->conn, MRConn_AuthCallback, conn, "AUTH %s", conn->ep.auth) ==
+  if (redisAsyncCommand(conn->conn, MRConn_AuthCallback, conn, "AUTH %s %s", clusterConfig.aclUsername, conn->ep.password) ==
       REDIS_ERR) {
     MRConn_SwitchState(conn, MRConn_ReAuth);
```

Comment:
I don't get it, looked at all the `MRConn_SendAuth` calls.
It seems like if it returns failure then we change the state to connecting, which is fine.
But internally in between we change the state to Reauth, which seems to start a timer.
Isn't there a better way of doing this?
Seems redundant to move to reauth state just to immediately move to connecting state.

Reply:
Where do we change to connecting?
I think we stay on `ReAuth`.

Reply:
See all the places `MRConn_SendAuth` gets called and what happens when it returns an error code to the caller.
The calling code proceeds to then change the start to connecting.
Unless I'm missing something.

Reply:
After discussing, we decided it is out of scope.